### ISSUE TITLE
High-grade vesicoureteral reflux

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6824]
**Preferred term label:**
High-grade vesicoureteral reflux

**Synonyms**

**Definition (free text, please give PubMed ID)**
Vesicoureteral reflux with dilation of the renal calyces (Grade III - V).
PMID: 18322164

**Parent term (use hpo.jax.org/app)**
Vesicoureteral reflux HP:0000076

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
added
